packages/auth/src/AuthStore.ts

```diff
@@ -131,9 +131,7 @@ export class AuthStore {
     return this.authClient?.logout({ returnTo: window.location.origin });
   }
 
-  async getTokenSilently(
-    options?: GetTokenSilentlyOptions
-  ): Promise<string | Error> {
+  async getTokenSilently(options?: GetTokenSilentlyOptions): Promise<string> {
```

Comment:
I was always curious about this - does TypeScript not care about typing thrown errors? I could've sworn it gave me problems when I removed the `| Error` but that might've been a different problem.

Reply:
yeah it's weird, I don't know of any way to specify the types that are thrown (probably would be hard to enforce in practice, since anything could happen further down the stack). If you care about it you just have to detect the type at runtime when you catch it (everything you catch defaults to `unknown`)

Reply:
Got it! Thank you so much for explaining!